adblockparser/parser.py

```diff
@@ -269,7 +269,7 @@ def rule_to_regex(cls, rule):
 
         # other | symbols should be escaped
         # we have "|$" in our regexp - do not touch it
-        rule = re.sub("(\|)[^$]", r"\|", rule)
+        rule = re.sub("(\|)(?!\$)", '\|', rule)
```

Comment:
:lipstick: I would add the `r` prefix to both strings instead of removing it, for future-proofing. But the suggested changes should work.

```suggestion
        rule = re.sub(r"(\|)(?!\$)", r'\|', rule)
```

Reply:
That's too long ago. I changed projects long ago. Do I have to accept the suggestions or could you do it as well?

Reply:
We can, don’t worry, and it’s not a deal breaker anyway. I’ll wait for a second approval, though.

PS: Thank you very much for your patch, I’m deeply sorry for the delay.